Fix the crosscompile patch to retain original behaviour, i.e. preventing
Darwin from failing to build.

--- 08_all_crosscompile.patch
+++ 08_all_crosscompile.patch
@@ -6,7 +6,7 @@
  
  PYTHON=		python$(EXE)
  BUILDPYTHON=	python$(BUILDEXE)
-+HOSTPYTHON=	$(BUILDPYTHON)
++HOSTPYTHON=	./$(BUILDPYTHON)
  
  # === Definitions added by makesetup ===
  
